src/app/src/pages/Profile.tsx

```diff
@@ -86,158 +118,228 @@ export default function Profile({ currentUser }: Props): JSX.Element {
     setFollowing(false);
   };
 
-  if (author !== undefined && currentUser!==undefined) {
+  if (author !== undefined && currentUser !== undefined) {
     return (
       <>
-      {open ? (
-        <Backdrop
-          sx={{
-            color: '#fff',
-            zIndex: (theme) => theme.zIndex.drawer + 1,
-            width: '100%',
-            height: '100%',
-            display: 'flex',
-            flexDirection: 'column',
-            alignItems: 'center',
-          }}
-          open={open}
-        >
-          <CloseRounded
-            onClick={handleClose}
+        {followerOpen ? (
+          <Backdrop
             sx={{
-              '&:hover': {
-                cursor: 'pointer',
-              },
-              marginBottom: '10px',
-              borderRadius: '100%',
-              border: '1px solid white',
+              color: "#fff",
+              zIndex: (theme) => theme.zIndex.drawer + 1,
+              width: "100%",
+              height: "100%",
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
             }}
-          />
-          <EditAuthor
-            data={currentUser}
-            handleAuthorsChanged={handleAuthorsChanged} 
-            handleClose={handleClose}
-          />
-        </Backdrop>
-      ) :(
-        <Box sx={{ height: window.innerHeight, width: window.innerWidth }}>
-          <Box style={{ height: '5%' }} sx={{ bgcolor: '#fff' }}>
-            <NavBar
-              items={[
-                {
-                  Text: '',
-                  handleClick: () => {
-                    console.log(1);
-                  },
+            open={followerOpen}
+          >
+            <CloseRounded
+              onClick={handleFollowerClose}
+              sx={{
+                "&:hover": {
+                  cursor: "pointer",
                 },
-              ]}
+                marginBottom: "10px",
+                borderRadius: "100%",
+                border: "1px solid white",
+              }}
             />
-          </Box>
-          <Box style={{ display: 'flex', height: '95%' }} sx={{ bgcolor: '#fff' }}>
-            <Box
-              boxShadow={5}
-              display="flex"
+            <FollowerList
+              data={currentUser}
```

Comment:
This will cause all profiles to only show the followers/following of the currentUser. You want to show the followers/following of the user's who's profile you are on.

Reply:
Changed!